dev-python/boost-histogram/boost-histogram-1.4.0.ebuild

```
@@ -0,0 +1,42 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="Python bindings for the C++14 Boost::Histogram library"
+HOMEPAGE="https://github.com/scikit-hep/boost-histogram"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-libs/boost[python,${PYTHON_USEDEP}]
```

```suggestion
	dev-libs/boost:=[python,${PYTHON_USEDEP}]
```
Probably this will need to be rebuilt when updating boost, have not checked though.

---

I don't think so since i thought it uses the boost python interface, but maybe i am wrong. I'll check that